.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: ruby
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y libvirt-dev
+  - gem install bundler --version $BUNDLER_VERSION
+install: bundle _${BUNDLER_VERSION}_ install
+script: bundle _${BUNDLER_VERSION}_ exec rspec --color --format documentation
+notifications:
+  email: false
+rvm:
+  - 2.0.0
+env:
+  global:
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  matrix:
+    - VAGRANT_VERSION=v1.5.4 BUNDLER_VERSION=1.5.3
```

Comment:
Since the README states it works on 1.5, 1.6 & 1.7 seems like it should be at least have a limited test on them and then if it breaks, when removing from the matrix can also update the README to remove the versions listed as suported.